src/dbus_api/blockdev/shared.rs

```diff
@@ -77,7 +77,18 @@ where
 /// Generate D-Bus representation of devnode property.
 #[inline]
 pub fn blockdev_devnode_prop(dev: &dyn BlockDev) -> String {
-    dev.devnode().user_path().display().to_string()
+    let pathbuf = match dev.user_path() {
```

Comment:
Can you explain this a little more? This is already applied to all of the `prop` methods as they're all only used in one place and I don't think it makes sense to require these to be a separate function for that reason. The only reason they're in their own method is because it makes it more readable.

Reply:
Ok. Let's leave it as is and stay consistent w/ the rest, now that I see the rule.

There's a bid of wierdness, though.

I did two experiments, both on release builds.

1. I removed the inline annotations on all dbus methods that had them, as an experiment, and the size of the executable seemed to decrease by about 3KiB. Odd, but not necessarily inexplicable.
2. Then I tried looking at objdump disassembly and that was perplexing as well. Basically I tried ```[inline(never)]``` and ```[inline(always)]``` on ```blockdev_devnode_prop```, and the disassembed code was _identical_ in the two cases.
3. And ```blockdev_devnode_prop``` seemed to be being invoked in two separate spots.

It's not a big deal, but it is perplexing. Also, the whole issue orthogonal to everything that this PR is about, so let's leave it entirely alone for now.

Reply:
Okay this is all good to know. I'll mark this as resolved for now.